tools/db_crashtest.py

```diff
@@ -836,15 +836,15 @@ def execute_cmd(cmd, timeout=None):
     return hit_timeout, child.returncode, outs.decode("utf-8"), errs.decode("utf-8")
 
 
-def exit_if_stderr_has_errors(stderr, print_stderr_separately=False):
+def print_stdout_and_exit_if_stderr_has_errors(stdout, stderr, print_stderr_separately=False):
```

Comment:
nit: the function prints both stdout and stderr but naming suggests only stdout.